prosper/common/prosper_logging.py

```diff
@@ -418,6 +402,7 @@ class DiscordWebhook(object):
 
     """
     __base_url = 'https://discordapp.com/api/webhooks/'
+    __webhook_url_format = __base_url + r"(\d+)/(\w+)"
```

Comment:
Heh, this is why I hate regex.  I thought I tried that and couldn't get it to work 😉